UI-Session-2-Assignment/function.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
1. When you click on navigation bar then it should automatically move to the particular section.
2. You need to give style to all fields belong to each section.

Reply:
Done